src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java

```diff
@@ -124,7 +125,7 @@ public void verifyColorAd(WebElement element, Color color, int durationSec, WebD
         PageObjectLogging.log("verifyColorAd", "Current time: " + currentTime + " seconds", true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "verifyColorAd" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)